Aspire.lutconfig

```diff
@@ -0,0 +1,6 @@
+<LUTConfig Version="1.0">
```

Comment:
Why was this added?

Reply:
Love LUT, but still struggle with this new version.